resources/assets/lib/chat/create-announcement.tsx

```diff
@@ -51,33 +50,47 @@ export default class CreateAnnouncement extends React.Component<Props> {
       <div className='chat-form'>
         <div className='chat-form__fields'>
           <div className='chat-form__title'>{trans('chat.form.title.announcement')}</div>
-          <InputContainer labelKey='chat.form.labels.name' model={this.model} modifiers='chat' name='name'>
+          <InputContainer
+            labelKey='chat.form.labels.name'
+            maxLength={maxLengths.name}
+            model={this.model}
+            modifiers='chat'
+            name='name'
+          >
             <input
               className='chat-form__input'
               defaultValue={this.model.inputs.name}
-              maxLength={50}
               name='name'
               onBlur={this.handleBlur}
               onChange={this.handleInput}
             />
           </InputContainer>
-          <InputContainer labelKey='chat.form.labels.description' model={this.model} modifiers='chat' name='description'>
+          <InputContainer
+            labelKey='chat.form.labels.description'
+            maxLength={maxLengths.description}
+            model={this.model}
+            modifiers='chat'
+            name='description'
+          >
             <input
               className='chat-form__input'
               defaultValue={this.model.inputs.description}
-              maxLength={255}
               name='description'
               onBlur={this.handleBlur}
               onChange={this.handleInput}
             />
           </InputContainer>
-          <InputContainer for='chat-form-users' labelKey='chat.form.labels.users' model={this.model} modifiers='chat' name='users'>
+          <InputContainer
+            labelKey='chat.form.labels.users'
+            model={this.model}
+            modifiers='chat'
+            name='users'
+          >
             <div className='chat-form__users'>
               <UserCardBrick user={core.currentUserOrFail} />
               {this.renderValidUsers()}
               <input
                 className='chat-form__input chat-form__input--users'
-                id='chat-form-users'
```

Comment:
this one here is actually required because there are also buttons inside the label...